pkg/remote/errors.go

```diff
@@ -17,7 +17,7 @@ package remote
 
 import "errors"
 
-// ErrorRequestInProgress is a sentinel value that indicates
+// ErrRequestInProgress is a sentinel value that indicates
 // resolving a remote file like a pipeline in a bundle or
 // a task in git hasn't completed yet.
-var ErrorRequestInProgress = errors.New("resource request in progress")
+var ErrRequestInProgress = errors.New("resource request in progress")
```

Comment:
So if we are "good citizens", we should alias the old one `ErrorRequestInProgress` and mark it as deprecated (`// Deprecated: use ErrRequestInProgress instead`)

Reply:
My 2c: the rename causes an easily-fixed compile-time error for anyone who was depending on the var, so deprecating prior to deletion just postpones the rename in dependent code.

That said, I'll add the "alias" if you feel strongly about it.

Reply:
Done.